.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug Fix
```

Comment:
Does the template automatically fill the labels for the issue? I'm wondering because we have no label named "Bug Fix" but we do have the "Bug" label.

Reply:
That's a good question. I don't know.

Reply:
Hey @driusan, apparently it does now as found [here](https://github.community/t5/How-to-use-Git-and-GitHub/Apply-labels-automatically-at-issue-creation/td-p/10719).
So maybe we should remove the "Fix" from "Bug Fix" for it to work.